model/technology.go

```diff
@@ -23,6 +23,8 @@ const (
 	TechnologyMariaDBFoundationMariaDB string = "MariaDBFoundation/MariaDB"
 	TechnologyPostgreSQLPostgreSQL     string = "PostgreSQL/PostgreSQL"
 	TechnologyOracleMySQL              string = "Oracle/MySQL"
+	TechnologyOracleOVM                string = "Oracle/OVM"
```

Comment:
I think the name is VM not OVM...
https://www.oracle.com/virtualization/technologies/vm/